tests/bud.bats

```diff
@@ -1620,15 +1621,24 @@ function _test_http() {
   gitrepo=github.com/projectatomic/nulecule-library
   run_buildah build $WITH_POLICY_JSON -t ${target} "${gitrepo}"
   run_buildah from ${target}
+  # Test where context is repourl.git#commit
+  gitrepo=github.com/projectatomic/nulecule-library#fa1c7025262f5d33ba6339dd70c1163c9c26108f
```

Comment:
Can this be changed to refer to a repository where the latest version of the default branch wouldn't have a Dockerfile, like the commit-and-subdirectory case below?

Reply:
Added test from local repo.